src/test/java/HttpVerifierTest.java

```diff
@@ -4,67 +4,155 @@
 
   HttpVerifier checker;
 
-  protected void setUp() {
-    checker = new HttpVerifier();
+  class InvalidMethodRequest implements HttpProtocal {
+    public String method() { return "XXXXX"; }
+    public String path() { return "/"; }
+    public String version() { return "HTTP/1.1"; }
+    public String requestLine() { return " "; }
   }
```

Comment:
I am going to try and refactor this test. So that instead of making a bunch of different mocks for each test. There will instead be a single Mock with a constructor that takes in method, path and version parameters. Then create an instance of that mock for each test passing in the values for that particular test.